### DISCUSSION TITLE
Common single sidebar for all pages

### DISCUSSION THREAD

**Discussion by abeka14** [writer/writer-framework#195] [Q&A]
Hi all. 

StreamSync is exactly what i was looking for. Thank you for developers. I want to build an app with several pages, and I want to navigate to those pages through buttons in left sidebar. StreamSync allows to use sidebar, but it puts it inside particular page. So, it is gone when you open a new page. Are there any ways to make a sidebar available in all pages? Thank you in advance!

**Comment by ramedina86** [comment]
Thanks @abeka14 . Right now it's not possible to repeat a component, you'd have to copy and paste it, which I agree isn't ideal if you'd like to have the exact same thing across pages. I'm converting this to an issue, and we'll develop this feature.

@FabienArcellier This is in line with the _Reuse component_ you proposed

**Reply by JE-VS** [reply]
Hi everybody, 

I'm actually trying to do exactly the same than @abeka14 
Thanks @ramedina86  and @FabienArcellier  for working on it! 

I was wondering, instead of "reusing" it, could it be possible to place the sidebar directly on the `root` component when we want it to be available for all pages ? It could also be a different component (a "global/root" sidebar), if it is easier.

**Comment by risedangel** [comment]
Maybe something like a "template page" feature can also be useful ?
